client/src/components/app/Users/ProjectsCarousel/index.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
```

Comment:
Let's refactor ProjectsCarousel and UsersCarousel into a single `Carousel` compoentn